frontend/src/metabase/parameters/components/ValuesSourceModal/ValuesSourceModal.unit.spec.tsx

```diff
@@ -393,6 +393,25 @@ describe("ValuesSourceModal", () => {
 
       expect(screen.getByRole("textbox")).toHaveValue("Gadget\nWidget");
     });
+    it("should render a hint about using models when labels are used", async () => {
+      await setup({
+        parameter: createMockUiParameter({
+          fields: [field1],
+          values_source_type: "static-list",
+          values_source_config: {
+            values: [["Gadget", "Label"], ["Widget"]],
+          },
+        }),
+      });
+
+      await userEvent.click(
+        screen.getByRole("radio", { name: "From connected fields" }),
+      );
+      await userEvent.click(screen.getByRole("radio", { name: "Custom list" }));
+
+      expect(screen.getByRole("textbox")).toHaveValue("Gadget, Label\nWidget");
+      expect(screen.getByText("do it once in a model")).toBeInTheDocument();
```

Comment:
I think we need 2 tests here:
- for `showMetabaseLinks: true` - we should assert that a link is rendered
- for `showMetabaseLinks: false` - we should assert that link is not rendered